pkg/kyc/component.go

```diff
@@ -408,6 +409,10 @@ func (c *component) validateUser(ctx context.Context, accessToken string, confRe
 	kcUser, err = c.keycloakClient.GetUser(accessToken, targetRealm, userID)
 	if err != nil {
 		c.logger.Warn(ctx, "msg", "Can't get user/accreditations", "err", err.Error())
+		var httpErr kc.HTTPError
+		if errors.As(err, &httpErr) && httpErr.HTTPStatus == 404 {
+			return errorhandler.CreateNotFoundError("user")
```

Comment:
I would use a c.logger.Info(...) here to tell the user does not exist and use c.logger.Warn(...) only in other cases.
Originally, we decided to log with Error(...) for errors that need an urgent action and log with Warn(...) when action has to be taken with no emergency.

Reply:
Okay thanks, I commited those changes. Did I understood correctly?